Textminer.py

```diff
@@ -10,20 +10,21 @@
 from pattern.en import *
```

Comment:
Thanks for removing all the .dat files :+1: I would like to see more of the comments address though